react/components/Buttons/AddToCart.js

```diff
@@ -0,0 +1,33 @@
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+import Button from 'vtex.styleguide/Button'
+import { injectIntl, intlShape } from 'react-intl'
+import ProductPrice from 'vtex.store-components/ProductPrice'
+
+class AddToCart extends Component {
+  static propTypes = {
+    total: PropTypes.number,
+    intl: intlShape.isRequired,
+  }
+
+  render() {
+    const intl = this.props.intl
+    const label = intl.formatMessage({ id: 'product-customizer.add-to-cart' })
```

Comment:
Use `<FormattedMessage />` component from `react-intl` to internationalize messages. See the [documentation](https://github.com/yahoo/react-intl/wiki/Components#string-formatting-components) for more details.

Reply:
fixed on commit https://github.com/vtex-apps/product-customizer/commit/db328474e5ad1ce958bfe33c8d362918a7c56924